src/System.Memory/src/System/Buffers/Text/Utf8Formatter/Utf8Formatter.Date.cs

```diff
@@ -78,7 +78,7 @@ public static partial class Utf8Formatter
         /// Formats a DateTimeOffset as a UTF8 string.
         /// </summary>
         /// <param name="value">Value to format</param>
-        /// <param name="buffer">Buffer to receive UTF8 string</param>
+        /// <param name="buffer">Buffer to write the UTF8-formatted value to</param>
         /// <param name="bytesWritten">Receives the length of the formatted text in bytes</param>
```

Comment:
Also here and elsewhere.

Reply:
Won't block on this: feel free to submit a PR for the XML docs - probably faster if you do it with the wording you feel works best.